src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Overall, good job. Naming of variables and methods are self explanatory.

Reply:
Thank you for the comment.

Reply:
Thank you for the kind comment.